README.md

```diff
@@ -1,40 +1,37 @@
+## What is the Eclipse Codewind sidecar container?
+The Codewind sidecar container includes the following responsibilities:
+- The sidecar deploys the Codewind server container.
+    - The sidecar renders the deployment and service templates and applies them with the `kubectl apply` command.
+    - When the workspace is shut down or deleted, the sidecar tears down Codewind and any deployed applications.
+- The sidecar sets up a reverse proxy for the Theia extension.
+    - Nginx is used for the proxy because it can handle both HTTP requests and socket.io.
+    - The Theia plug-in communicates with the reverse proxy, which then forwards requests to Codewind. This chain of communication avoids the addition of code in the Theia plug-in to discover and manage the connection to Codewind.
+- The sidecar runs the `filewatcherd` daemon to track user code changes.
+    - The `filewatcherd` daemon watches for changes in each user's project and communicates with Codewind, letting it know to start a build if required.
+    - For more information on `filewatcherd`, see [eclipse/codewind-filewatchers](https://github.com/eclipse/codewind-filewatchers).
 
-The Codewind sidecar container has a number of responsibilities:
-- Deploying the Codewind server container
-    - It renders the deployment and service templates and `kubectl apply`'s them.
-    - When the workspace is shut down or deleted, the sidecar will tear down Codewind, as well as any deployed applications.
-- Setting up a reverse proxy for the Theia extension
-    - We use Nginx for the proxy, as it can handle both HTTP requests and socket.io
-    - The Theia plugin communicates with the reverse proxy (which then forwards requests to Codewind). This prevents us from having to add code in the Theia plugin to discover and manage the connection to Codewind
-- Running the `filewatcherd` daemon, to track user code changes
-    - `filewatcherd` watches for changes in each of the user's projects, and communicates with Codewind, letting it know to start a build (if required)
-    - For more details on filewatcherd, see https://github.com/eclipse/codewind-filewatchers
 
-
-## Development
+## Developing
 
 ### Prerequisites
 
-- Docker 17.05 or higher
+- Install Docker 17.05 or later.
 
-### Build
+### Building
 
 To build the sidecar image, run `./build.sh`.
 
-### Deployment
+### Deploying
 
-Follow the readme at (https://github.com/eclipse/codewind-che-plugin/tree/master/scripts) for deployment instructions.
+For deployment instructions, see the README.md file at [eclipse/codewind-che-plugin](https://github.com/eclipse/codewind-che-plugin/tree/master/scripts).
 
 ## Contributing
-We welcome submitting issues and contributions.
-1. [Submitting bugs](https://github.com/eclipse/codewind-che-plugin/issues)
-2. [Contributing](CONTRIBUTING.md)
-
-## License
```

Comment:
Did you intentionally remove the license link?

Reply:
Is there a way we can remove the link at the bottom and put it with the other link at the top? That way all the license info is together?